core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -166,6 +166,7 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_SQL_MAX_RECOMPILE_ATTEMPTS("cairo.sql.max.recompile.attempts"),
     CAIRO_ATTACH_PARTITION_SUFFIX("cairo.attach.partition.suffix"),
     CAIRO_ATTACH_PARTITION_COPY("cairo.attach.partition.copy"),
+    CAIRO_COMMIT_LATENCY("cairo.commit.latency"),
```

Comment:
This property should be documented in the defaults in built-in `server.conf`.

Reply:
this is what i referred to in the "side effects" section

Reply:
I see. It'll certainly make the config much easier to deal with. Good idea!